src/presence/PresenceHeartbeat.js

```diff
@@ -1,82 +1,48 @@
 // @flow strict-local
-import { PureComponent } from 'react';
-import type { ComponentType } from 'react';
+import * as React from 'react';
+import { useEffect } from 'react';
```

Comment:
nit: I think we've lately preferred using `React.useEffect` rather than importing `useEffect` separately?